src/pages/Home.jsx

```diff
@@ -0,0 +1,38 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { getCountries } from '../redux/countries/countriesSlice';
+import Country from '../components/country';
+import { getCountryStates } from '../redux/countries/statesSlice';
+
+const Home = () => {
+  const dispatch = useDispatch();
+
+  const handleClick = (country) => {
+    dispatch(getCountryStates(country));
+  };
+
+  useEffect(() => {
+    dispatch(getCountries());
+  }, [dispatch]);
+
+  const countries = useSelector((store) => store.countries.countries);
+  return (
+    <main className="gen container container-md-fluid">
```

Comment:
- [ ] Place place these cards together as seen in the original design. Currently, you have left spaces between each card
### Your Implementation
![image](https://github.com/Gh-Reza/weather-app/assets/45460192/88c4e12a-2ef5-425e-906e-7433dca24adc)
### Required
![image](https://github.com/Gh-Reza/weather-app/assets/45460192/3d211dfc-dbc2-4a5f-9adf-123a50a40f03)